src/main/java/ibas/inchelin/domain/notice/entity/Notice.java

```diff
@@ -12,16 +12,17 @@
 @Entity
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
-public class Notice extends BaseTimeEntity {
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+public class Notice {
```

Comment:
데이터의 생성 및 수정 시간을 기록하기 위해 BaseTimeEntity를 상속받았으나, 해당 부분이 제거되었으므로 다시 추가해 주세요.
```suggestion
public class Notice extends BaseTimeEntity {
```